File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/HarvestsUpdateHarvestsLocationRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class HarvestsUpdateHarvestsLocationRequestItem(
    @JsonProperty("ActualDate")
    val actualDate: String? = null,
    @JsonProperty("DryingLocation")
    val dryingLocation: String? = null,
    @JsonProperty("DryingSublocation")
    val dryingSublocation: String? = null,
    @JsonProperty("HarvestName")
    val harvestName: String? = null,
    @JsonProperty("Id")
    val id: Int? = null
)

File: sdks/thunkmetrc-go/wrapper/models/harvests_update_harvests_location_request_item.go
package models

type HarvestsUpdateHarvestsLocationRequestItem struct {
    ActualDate string `json:"ActualDate,omitempty"`
    DryingLocation string `json:"DryingLocation,omitempty"`
    DryingSublocation string `json:"DryingSublocation,omitempty"`
    HarvestName string `json:"HarvestName,omitempty"`
    Id int `json:"Id,omitempty"`
}
